src/components/ui/QuickSearchButton.stories.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import { storiesOf } from '@storybook/react'
+import { action } from '@storybook/addon-actions'
+
+import QuickSearchButton from './QuickSearchButton'
+
+const Body = ({ children }) => (
+  <div style={{ width: '200px', padding: '10px 0', border: '1px solid black' }}>
```

Comment:
Since this is a default showcase (not a debug one), lets remove extra visual parts to not cause confusion. Under the Debug category this would be fine, but other than that, some Storybook users (as the designers) will be confused if they see something that's not expected in the design, like the border in this case.

```suggestion
  <div style={{ width: '64px', padding: '10px 0' }}>
```